Solutions/IoTOTThreatMonitoringwithDefenderforIoT/Workbooks/IoTOTThreatMonitoringwithDefenderforIoT.json

```diff
@@ -91,10 +93,11 @@
               ],
               "showDefault": false
             },
+            "defaultValue": "value::all",
             "queryType": 1,
             "resourceType": "microsoft.resourcegraph/resources",
             "value": [
-              "value::all"
+              "/subscriptions/81de6a21-8799-4bb5-a28c-cbc292f8e023/resourceGroups/amamit-test2/providers/Microsoft.OperationalInsights/workspaces/Amit"
```

Comment:
Hi @amitcohen25 Please remove these hardcoded values. Because of these validations are failing.
Thanks

Reply:
Hi @NikTripathi , thanks for highlighting this. I fixed it

Reply:
@NikTripathi The output you shared (missing parameters) is fine. You should first select a record from the table and then it will work